contrib/select_untagged.lua

```diff
@@ -0,0 +1,65 @@
+--[[
+    This file is part of darktable,
+    copyright (c) 2017 Jannis_V
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    darktable is distributed in the hope that it will be useful,
+     but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[
+Enable selection of untagged images (darktable|* tags are ignored)
+]]
+
+local dt = require "darktable"
+require "official/yield"
+local gettext = dt.gettext
+
+dt.configuration.check_version(...,{3,0,0},{4,0,0},{5,0,0})
+
+-- Tell gettext where to find the .mo file translating messages for a particular domain
+gettext.bindtextdomain("select_untagged",dt.configuration.config_dir.."/lua/locale/")
```

Comment:
Do you have any translations, that you could add to the pr?

Reply:
Not yet, but I will add english and french translations.

Reply:
I have created a file containing a german translation (locale/de_DE/LC_MESSAGES/select_untagged.po).
The translation is not applied in darktable 2.2.5, gettext.dgettext always returns the original message, the other GUI elements which are not added by scripts are in german as expected.
According to inotifywait, the file is not even accessed by darktable, same with the translations of other scripts.
Does the translation in scripts work for you in 2.2.5?

Reply:
Should I include the .mo file, or is it left to the user of the script to build it?

Reply:
As we don't have a build system for the mo file I would add them.

Reply:
I have added them to my branch.

Reply:
then please make a new pull request